unit/catch_example.cpp

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************\
+
+ Module: Example Catch Tests
+
+ Author: DiffBlue Limited. All rights reserved.
+
+\*******************************************************************/
+
+#include <catch.hpp>
+
+unsigned int Factorial( unsigned int number ) {
+  return number > 1 ? Factorial(number-1)*number : 1;
+}
+
+///
+/// This is an example unit test to demonstrate the build system and the
+/// catch unit test framework. The source code is taken from the documentation
+/// of catch.
+///
```

Comment:
This comment block is not in a standard format. If you want to include the example you probably need to DiffBlue-ise it.